src/internal_rep.rs

```diff
@@ -1747,7 +1755,8 @@ impl<'a> FunctionInfo<'a> {
             original_body: &[], // Unused after validation
             body: Some(derived_body),
             declaration_file: file,
-            is_associated_call: false, // I *think* the behavior should be that this is the true if it's true for *any* parent.  TODO: check/document that
+            is_associated_call: derived_is_associated_call,
```

Comment:
Hmm, I'm not sure but I think it would make more sense to consider `@associated_call` as part of the function signature and then forbid function composition if they don't all have this annotation.

Reply:
Yeah, I think you're right.  I'll see about changing that for the next version of the PR.